src/ec/packages/ec-component-site-header-harmonised/ecl-site-header-harmonised.html.twig

```diff
@@ -171,125 +171,138 @@
-            href="{{ _login_toggle.href_logged }}"
-            aria-controls="{{ _login_box.id }}"
-            data-ecl-login-toggle
-            aria-expanded="false"
-          >
-            {% include '@ecl-twig/ec-component-icon/ecl-icon.html.twig' with {
-              icon: {
-                path: _icon_file_path,
-                type: 'general',
-                name: 'logged-in',
-                size: 'm'
-              },
-              extra_classes: 'ecl-site-header-harmonised__icon'
-            } only %}
-             {{- login_toggle.label_logged -}}
-            {% include '@ecl-twig/ec-component-icon/ecl-icon.html.twig' with {
-              icon: {
-                path: _icon_file_path,
-                type: 'ui',
-                name: 'corner-arrow',
-                size: 'xs'
-              },
-              extra_classes: 'ecl-site-header-harmonised__login-arrow'
-            } only %}
-          </a>
-          <div
-            id="{{ _login_box.id }}"
-            class="ecl-site-header-harmonised__login-box"
-            data-ecl-login-box
-          >
-            <p class="ecl-site-header-harmonised__login-description">
-              {{- _login_box.description|raw -}}
-            </p>
-            <hr className="ecl-site-header-harmonised__login-separator" />
-            {% include '@ecl-twig/ec-component-link/ecl-link.html.twig' with {
-              link: {
-                label: _login_box.label,
-                path: _login_box.href,
-                type: "standalone"
-              }
-            } only %}
-          </div>
+        {% if group != 'group3' %}
+          {% if site_name is not empty %}
+            <div class="ecl-site-header-harmonised__site-name">{{ site_name }}</div>
+          {% endif %}
+        </a>
         {% else %}
-          <a
-            class="ecl-link ecl-link--standalone ecl-site-header-harmonised__login-toggle"
-            href="{{ _login_box.href_not_logged }}"
-          >
-            {% include '@ecl-twig/ec-component-icon/ecl-icon.html.twig' with {
-              icon: {
-                path: _icon_file_path,
-                type: 'general',
-                name: 'log-in',
-                size: 'm'
-              },
-              extra_classes: 'ecl-site-header-harmonised__icon'
-            } only %}
-            {{- login_toggle.label_not_logged -}}
-          </a>
+        </a>
+        {% if site_name is not empty %}
+          <div class="ecl-site-header-harmonised__site-name">{{ site_name }}</div>
         {% endif %}
-        </div>
       {% endif %}
-      {% if _language_selector is not empty %}
-        <a
-          class="ecl-link ecl-link--standalone ecl-site-header-harmonised__language-selector"
-          href="{{ _language_selector.href }}"
-          data-ecl-language-selector
-        {% if _has_overlay is defined %}
-          aria-controls="language-list-overlay"
-          aria-expanded="false"
+
```

Comment:
General issue in the template now, alignment, indentation. I know it's hard to keep the indentation especially when we use logic in the templates, but we should try to preserve the indentation of the html tags, a closing tag has to correspond to the opening tag, we have two spaces indentation for all the tags, if(s) can be adapted, indentation of the html tags no.